src/mui/layout/Layout.js

```diff
@@ -10,8 +10,8 @@ import Menu from './Menu';
 
 injectTapEventPlugin();
 
-const Layout = ({ isLoading, children, route, title }) => {
-    const Title = <Link to="/" style={{ color: '#fff', textDecoration: 'none' }}>{title}</Link>;
+const Layout = ({ isLoading, children, route, title, basePath }) => {
+    const Title = <Link to={`/${basePath}`} style={{ color: '#fff', textDecoration: 'none' }}>{title}</Link>;
```

Comment:
We can't add the `basePath` to all `<Link>` calls. Also, we can't add the `basePath` as parameter to all components that require a link. It'd be better to add an `<AdminLink>` component taking care of it automatically, connected to the store, where we'd put the `basePath`.